### ISSUE TITLE
delayInSession not working

### ISSUE THREAD

**Issue by francoism90** [cyrildewit/eloquent-viewable#205]
- Eloquent Viewable: 4.1
- Laravel Version: 6.18.0
- PHP Version: 7.4
- Database Driver & Version: MariaDB 10.4.12

### Steps To Reproduce:

```
    public function recordView(string $collection = null): void
    {
        views($this)
            ->delayInSession(now()->addHour())
            ->overrideIpAddress(request()->ip())
            ->overrideVisitor(auth()->user()->id ?? 0)
            ->collection($collection)
            ->record();
    }
```

```
IP address: 192.168.1.100
UserID: 1
Model: Post - ID: 7
```

#### Expected behavior:

Do not save/insert in database when being run multiple times in specified cooldown.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/5028905/75989336-6010d080-5ef3-11ea-91d8-1eecc99da3a6.png)

**Comment by francoism90** [comment]
@cyrildewit I'm using JWT (https://github.com/tymondesigns/jwt-auth) and my session ID keeps changing. Is this the reason it keeps saving records to the database?

**Comment by cyrildewit** [comment]
Yes, that's probably the case. The package uses the session to keep track of these delays.

**Comment by francoism90** [comment]
Does your package support SPA's? I'm thinking of moving to Airlock, but I haven't tested this yet and don't know if this solves my problem.

**Comment by francoism90** [comment]
@cyrildewit I'm testing the upcoming v5.0 release including Airlock, and so far it seems to correctly record the views with those that include a cooldown! :D

**Comment by cyrildewit** [comment]
Great! I am going to think a little bit more about the extendability of the cooldown functionality. I already have an idea to overwrite it by providing a callable, but I want to take a good look at it before adding it.

I've added it to the v5 roadmap.

**Comment by francoism90** [comment]
Thanks for your work and help on this. :)

Do you have any ETA for V5? Airlock (and other packages I'm using as well)
depend on Laravel 7 unfortunately. Could you maybe release an alpha or beta
version? At the moment I've included dev-release 5.0 as version, but that
isn't considered best practice. If you mark is as unstable I do think
people will understand things may be broken or change in the near future.

vriendelijke groet,

François Menning
